### PR TITLE
Daisies & Ivy

### PR DESCRIPTION
**Pull request by careeoki**
Two new carpet-type blocks, this time they're plants. They both can help to make biomes more detailed, and of course are nice to build with.

image
image
Item textures:
image

They use a new plane model cannot be selected from the side (because... it doesn't have one.) I don't actually think this is a problem, but i wanted to mention it.
image

### CODE REVIEW

**Comment by IntegratedQuantum** [comment]
To be honest, I think the ivy could be better.
Usually ivy leaves have more of a glossy texture. So maybe adding some reflectivity could help.

**Comment by careeoki** [comment]
I tried a reflectivity map and it looked odd.
I made the leaves brighter, so they 'pop' more, and made the palette more dynamic.

image

image
Let me know what you think.

**Review by IntegratedQuantum — COMMENTED** [review]
Sad that reflection doesn't work, but I guess it is kind of an odd effect anyways.

The new version looks better, makes the leaves more distinct. (though it does make the texture a little bit more repetitive, but that's nothing to worry about, since this could be easily fixed by random texture variants)

If you don't mind, I'd also like to see how a version with a bit more saturation would look.

File: assets/cubyz/blocks/daisies.zig.zon [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	.hardness = 0.1,
+	.drops = .{
+		.auto,
+	},
+	.degradable = true,
+	.collide = false,
+	.alwaysViewThrough = true,
+	.absorbedLight = 0x121012,	
+	.model = "cubyz:plane",
+	.rotation = .carpet,
```
[the remark this answers is not in the document]
Comment (careeoki) [reply assets/cubyz/blocks/daisies.zig.zon:12]: You can place them on leaves for blooming flowers, or to decorate a trellis, etc. But yeah, you wouldn't see that in natural generation.

**Comment by careeoki** [comment]
Reflection works best for really shiny things like metal or glass.

>  (though it does make the texture a little bit more repetitive, but that's nothing to worry about, since this could be easily fixed by random texture variants)

I still have to fix some tiling errors as well.

With more saturation:
I wouldn't want to overdo it.
image
image

**Comment by IntegratedQuantum** [comment]
> With more saturation:
I wouldn't want to overdo it.

Hmm, yeah I think I understand what you mean.

Would you mind sending me the latest texture, so I can do some more experiments on my own with different colors?

**Comment by careeoki** [comment]
Sure, I pushed it to my branch.

**Comment by IntegratedQuantum** [comment]
Thank you, I played around a bit, and I think the texture you made is good.

One more thing though: Why not make the gap between to the background block smaller? It kind of bothers me that there is such a large gap between the ivy and the wall.
For example at one quarter of the distance, it's still far enough appear to appear spatially different, but close enough that you don't get these large gaps:
Screenshot at 2024-11-27 21-58-29
Screenshot at 2024-11-27 21-57-16

**Comment by IntegratedQuantum** [comment]
Yeah, I get that (though it would be better to actually have stems below the flowers, but that is a different story)

**Pull request merged**